src/rebar_utils.erl

```diff
@@ -179,19 +179,6 @@ expand_code_path() ->
                            end, [], code:get_path()),
     code:set_path(lists:reverse(CodePath)).
 
-vcs_vsn(Config, Vsn, Dir) ->
-    Key = {Vsn, Dir},
-    Cache = rebar_state:get(Config, vsn_cache, dict:new()),
```

Comment:
Dropping cruft. It isn't used, and if we do need a cache of vsns it should be done simply by updating the vsn in the `app_info` record.